cookbooks/aws-parallelcluster-slurm/templates/default/slurm/head_node/update_munge_key.sh.erb

```diff
@@ -15,12 +15,31 @@ MUNGE_USER="<%= @munge_user %>"
 MUNGE_GROUP="<%= @munge_group %>"
 SHARED_DIRECTORY_COMPUTE="<%= @shared_directory_compute %>"
 SHARED_DIRECTORY_LOGIN="<%= @shared_directory_login %>"
+CONFIG_PROCESS="False"
+
+while getopts "c:" opt; do
+  case $opt in
+    c)
+      if [[ $OPTARG != "True" && $OPTARG != "False" ]]; then
+        echo "Invalid value for -c: $OPTARG. Expected 'True' or 'False'."
+        exit 1
+      fi
+      CONFIG_PROCESS=$OPTARG
+      ;;
+    *)
+      echo "Usage: $0 [-c True/False]" >&2
+      exit 1
+      ;;
+  esac
+done
 
-# Check compute fleet status
-compute_fleet_status=$(get-compute-fleet-status.sh)
-if ! echo "$compute_fleet_status" | grep -q '"status": "STOPPED"'; then
-    echo "Compute fleet is not stopped. Please stop it before updating the munge key."
-    exit 1
+if [ "$CONFIG_PROCESS" == "False" ]; then
+  # Check compute fleet status
+  compute_fleet_status=$(get-compute-fleet-status.sh)
+  if ! echo "$compute_fleet_status" | grep -q '"status": "STOPPED"'; then
+      echo "Compute fleet is not stopped. Please stop it before updating the munge key."
+      exit 1
+  fi
```

Comment:
Where is the check on the login nodes status? Is it in a different PR (that depends on this one)?

Reply:
It's in this other PR: https://github.com/aws/aws-parallelcluster-cookbook/pull/2473/files#diff-449b77b3cae720dd4ab62600d04ca4262e20adb4e072ffb30de6de2580bc3bd2

Reply:
Yes. I won't miss it.